src/main/resources/kobiton/ExecuteScript.py

```diff
@@ -0,0 +1,80 @@
+import urllib2
+import json
+import copy
+
+username = kobitonServer['username']
+apiKey = kobitonServer['apiKey']
+
+remoteServer = kobitonServer['remoteServer']
+
+defaultDeviceOrientation = kobitonServer['deviceOrientation']
+defaultCaptureScreenshots = kobitonServer['captureScreenshots']
+
+EXECUTE_TEST_URL = remoteServer + '/submit'
+
+def merge_devices():
+    result = []
+    for udid in privateDevices:
+        result.append({
+            'udid': udid
+        })
+
+    for id in kobiDevices:
+        params = kobiDevices[id].split(' | ')
+        if params[3] == 'cloudDevices':
+            result.append({
+                'deviceName': params[0],
+                'platformName': params[1],
+                'platformVersion': params[2]
+            })
+        elif (id in privateDevices) == False:
+                result.append({
+                    'udid': id
+                })
+        
+    return result
+
+def send_request(devicesList):
+    if devicesList == []:
+        print 'No device to execute tests.'
```

Comment:
Just have one concern do we need to stop the release when no devices to execute test ?

Reply:
If there's no device, the task will be marked as Complete, since we have no output. => the release wont stop